webview/src/shared/components/sectionContainer/InfoTooltip.tsx

```diff
@@ -1,47 +1,52 @@
-import React from 'react'
+import React, { PropsWithChildren } from 'react'
 import cx from 'classnames'
 import { PlotsSection } from 'dvc/src/plots/webview/contract'
 import { SetupSection } from 'dvc/src/setup/webview/contract'
 import styles from './styles.module.scss'
-import { SectionDescription } from './SectionContainer'
+import { SectionDescription } from './SectionDescription'
 import Tooltip from '../tooltip/Tooltip'
+import { Info, PassFilled, Error, Warning } from '../icons'
 import { Icon } from '../Icon'
-import { Info, PassFilled, Error } from '../icons'
 
 export enum TooltipIconType {
   PASSED = 'pass-filled',
   INFO = 'info',
-  ERROR = 'error'
+  ERROR = 'error',
+  WARNING = 'warning'
 }
 
 const tooltipIcons = {
-  [TooltipIconType.PASSED]: PassFilled,
-  [TooltipIconType.INFO]: Info,
-  [TooltipIconType.ERROR]: Error
+  [TooltipIconType.PASSED]: {
+    className: styles.completedIcon,
+    icon: PassFilled
+  },
+  [TooltipIconType.INFO]: { className: undefined, icon: Info },
+  [TooltipIconType.ERROR]: { className: styles.errorIcon, icon: Error },
+  [TooltipIconType.WARNING]: { className: styles.warningIcon, icon: Warning }
 }
 
-export const InfoTooltip: React.FC<{
-  sectionKey: PlotsSection | SetupSection
-  icon?: TooltipIconType
-}> = ({ icon = TooltipIconType.INFO, sectionKey }) => {
+export const InfoTooltip: React.FC<
+  PropsWithChildren<{
+    sectionKey: PlotsSection | SetupSection
+    icon?: TooltipIconType
+  }>
+> = ({ icon = TooltipIconType.INFO, sectionKey, children }) => {
   const infoIcon = (
```

Comment:
[N] I think leaving this named `infoIcon` is misleading because it is no longer always the information icon. Maybe `indicatorIcon` would be a better name.

Reply:
Good idea! We actually use the term info in multiple places regarding the component, data idea, component name, class, etc. I'll rename the different places in a followup :)